tasks/service.yml

```diff
@@ -1,13 +1,21 @@
 ---
 
+- name: Copy setenv script
```

Comment:
Is this needed? It runs fine for me in my testing without setting this on Cent 7.

Probably doesn't hurt to add, but just wondering why it was added.

Reply:
Although Ansible said all the tasks completed successfully, Blazegraph and Fedora wouldn't start without it.

Reply:
I wonder if we are masking some issues with the Blazegraph and Fedora roles by setting `JAVA_HOME` here then? Since it seems like Karaf starts fine without it...

Reply:
I built the tasks based on the instructions in the [Karaf Quick Guide](https://karaf.apache.org/manual/latest/) which included this script. I simply included it because they did.

Reply:
Okay cool. 👍🏻 

Just wanted some detail why its there.